electrumsv/tests/test_nodeapi.py

```diff
@@ -556,6 +556,21 @@ def get_visible_accounts() -> list[StandardAccount]:
     ("createrawtransaction", [ [], {}, 0xFFFFFFFF+1  ], RPCError.INVALID_PARAMETER,
         "Invalid parameter, locktime out of range"),
 
+    ## ``getbalance``: ``minconf`` parameter
```

Comment:
What about "Invalid parameter, unexpected utxo type: "

Reply:
Have added a new test for this now